iOS/Controllers/CalendarViewController.swift

```diff
@@ -82,15 +82,56 @@ class CalendarViewController: DayViewController {
 
         return self.calendarEvents.filter(inInterval: interval).map { $0.calendarKitEvent }
     }
+
+    override func dayViewDidSelectEventView(_ eventView: EventView) {
+        guard let eventDescriptor = eventView.descriptor as? Event else { return }
+        guard let userInfo = eventDescriptor.userInfo as? [String: String] else { return }
+        guard let id = userInfo["objectID"] else { return }
+        guard let event = self.calendarEvents.first(where: { $0.id == id }) else {
+            return
+        }
+
+        func clampMax(_ a: CGFloat, _ b: CGFloat) -> CGFloat { return min(a, b) }
+
+        guard let popupEvent = R.storyboard.calendar.popupEvent() else { return }
+        let preferedHeight = popupEvent.preferredContentHeight(width: 500, for: event.description ?? "")
+        let height = clampMax(preferedHeight, 500)
+        popupEvent.preferredContentSize = CGSize(width: 500, height: height)
+        popupEvent.event = event
+        popupEvent.modalPresentationStyle = .popover
+
+        self.present(popupEvent, animated: true)
+
+        let popup = popupEvent.popoverPresentationController
+        popup?.permittedArrowDirections = [.left, .down, .up]
+        popup?.sourceView = self.view
+        popup?.delegate = self
+
+        var rect = eventView.convert(eventView.bounds, to: self.view)
+        rect.size.width = 200 // makes the popover show on top of the event, since so much horizontal space 
```

Comment:
I don't really get the comment. Is this workaround needed? Why not remove `.left` from `permittedArrowDirections`?

Reply:
By default, the popover will never cover part of the `rect` assigned to the popover. And because there is a lot of wasted space on calendar EventView on the right, I want the popover to sit there, with it's arrow facing left, instead of being on top or bottom of the view.

Reply:
Can you please add some screenshot to visualize this issue?

Reply:
Here is what happens without the manual setting of the width:
![Simulator Screen Shot - iPad Air 2 - 2019-06-26 at 10 59 11](https://user-images.githubusercontent.com/2438156/60166760-0acbdd80-9802-11e9-86fd-b56328f0d127.png)
![Simulator Screen Shot - iPad Air 2 - 2019-06-26 at 10 59 17](https://user-images.githubusercontent.com/2438156/60166762-0acbdd80-9802-11e9-9007-f756a276bc4d.png)

Here is what setting the size do: 
![Simulator Screen Shot - iPad Air 2 - 2019-06-26 at 11 02 26](https://user-images.githubusercontent.com/2438156/60166798-1c14ea00-9802-11e9-930c-6565f3d0f882.png)
![Simulator Screen Shot - iPad Air 2 - 2019-06-26 at 11 02 33](https://user-images.githubusercontent.com/2438156/60166800-1c14ea00-9802-11e9-8aca-a7ac7794dad5.png)

Reply:
Okay I see your point 👍